pom.xml

```
@@ -52,10 +52,13 @@ under the License.
 
 	<modules>
 		<module>flink-connector-opensearch-base</module>
+		<module>flink-connector-opensearch2</module>
+		<module>flink-connector-opensearch2-e2e-tests</module>
+		<module>flink-sql-connector-opensearch2</module>
 	</modules>
 
 	<properties>
-		<flink.version>1.20.0</flink.version>
+		<flink.version>2.0.0</flink.version>
 		<commons-compress.version>1.26.1</commons-compress.version>
 		<jackson-bom.version>2.16.2</jackson-bom.version>
 		<junit5.version>5.10.2</junit5.version>
```

```suggestion
		<jackson-bom.version>2.18.2</jackson-bom.version>
		<junit5.version>5.11.4</junit5.version>
```
how about bumping it to at least the same that is for flink 2.0?